docs/UserGuide.adoc

```diff
@@ -156,6 +156,11 @@ Views all details of the 1st person in the results of the `find` command.
 Clears all entries from the address book. +
 Format: `clear`
 
+== Sorting all entries : `sort`
```

Comment:
Good job for updating the User Guide!

Would be nice if you could give an example. Can possibly show the output of the `list` command followed by that of the `sort` command.